js/mainjs/trayMenu.js

```diff
@@ -0,0 +1,22 @@
+'use strict';
+
+const Electron = require('electron');
+const Menu = Electron.Menu;
+
+module.exports = function (window) {
+	// Template for Sia-UI tray menu.
+	var menutemplate = [
+		{
+			label: 'Show Sia',
+			click: function () { window.show(); }
+		},
+		{ type: 'separator' },
+		{
+			label: 'Hide Sia',
+			click: function () { window.hide(); }
```

Comment:
could these just be `window.show` and `window.hide`?
although, even if it's possible, I'm not sure it's best practice.